apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -37,28 +39,26 @@ const createOnboarding = async ({ keystone, user }) => {
  */
 const cleanEmailForAlreadyExistingUserWithGivenEmail = async ({ email, userIdToExclude, context }) => {
     if (!email) throw new Error('email argument is not specified')
-    const [ existingUser ] = await getItems({
-        ...context,
-        listKey: 'User',
-        where: { email, id_not: userIdToExclude },
-        returnFields: 'id type name email phone importId importRemoteSystem',
-    })
+
+    const [ existingUser ] = await User.getAll(context, { email, id_not: userIdToExclude })
     if (existingUser && existingUser.id !== userIdToExclude) {
-        await updateItem({
-            listKey: 'User',
-            item: {
-                id: existingUser.id,
-                data: {
-                    email: null,
-                    ...dvSenderFields,
-                },
-            },
-            returnFields: 'id',
-            ...context,
+        await User.update(context, existingUser.id, {
+            email: null,
+            ...dvSenderFields,
         })
     }
 }
 
+const registerIdentity = async ({ context, user, userInfo }) => {
+    await registerUserExternalIdentity(context, {
+        ...dvSenderFields,
+        user: { id: user.id },
+        identityId: userInfo.importId,
+        identityType: SBBOL_IDP_TYPE,
+        meta: {},
```

Comment:
should we put some userInfor to the meta?

Reply:
It going to be useless, since all data in userInfo, for sbbol integration, are placed in user & company.
In general the 'meta' field describes external user meta info,  that can be useful by userExteranlIdentity provider itself. (since meta field structure is strictly depends by external identity provider)
That means for sbbol case - we don't need to save such data. More over - the data for sbbol case already saved in user.meta

Reply:
It looks strange because this meta from extarnal identity provider